tests/element_test_vectors.rs

```diff
@@ -249,13 +230,15 @@ trait ElementApi {
         Ok(())
     }
 
-    fn read_file(reader: &Self::ReaderApi, file_name: &str) -> IonResult<Vec<Element>> {
-        // TODO have a better API that doesn't require buffering into memory everything...
```

Comment:
🗺️ Removing this old comment; he native reader implementations don't have to load the entire input into memory. 🎉

Reply:
🎉